src/js/views/scores.js

```diff
@@ -20,6 +20,18 @@ define('views/scores',[
                 $scope.rev = (String($scope.sort) === String(col)) ? !$scope.rev : !!defaultSort;
                 $scope.sort = col;
             };
+
+            $scope.sortIcon = function(col){
+                if(String($scope.sort)!== String(col)){//col and $scope.sort can be arrays, and so this is a quick and dirty way to check for equality
```

Comment:
Adding an external package sounds right here. If you can do that, it would be great